docs.joern.io/content/frontends/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+id: frontends 
+title: Frontends
+bookCollapseSection: true
+weight: 135
+---
```

Comment:
Could we populate this with a short intro into what the frontends are, and how to invoke frontends with `joern-parse`? Also, would be good to note where `x2cpg` args come in, and what `--frontend-args` do.